source/administration/mmctl-cli-tool.rst

```diff
@@ -2355,36 +2356,65 @@ mmctl permissions add
    --local                       allows communicating with the server through a unix socket
    --strict                      will only run commands if the mmctl version matches the server one
 
-mmctl permissions role assign
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+mmctl permissions remove
+^^^^^^^^^^^^^^^^^^^^^^^^
 
 **Description**
 
-Assign users to a role by username (only available in Enterprise Edition E10 and E20).
+  Remove one or more permissions from an existing role (only available in E10 and E20).
 
 **Format**
 
 .. code-block:: sh
 
-  mmctl permissions role assign [role_name] [username...] [flags]
+      mmctl permissions remove [role_name] [permission...] [flags]
 
 **Examples**
 
 .. code-block:: sh
 
-    # Assign users with usernames 'john.doe' and 'jane.doe' to the role named 'system_admin'.
-    permissions assign system_admin john.doe jane.doe
-    
-    # Examples using other system roles
-    permissions assign system_manager john.doe jane.doe
-    permissions assign system_user_manager john.doe jane.doe
-    permissions assign system_read_only_admin john.doe jane.doe
+      permissions remove system_user list_open_teams
 
 **Options**
 
 .. code-block:: sh
 
-  -h, --help   help for assign
+     -h, --help   help for remove
+
+**Options inherited from parent commands**
+
+.. code-block:: sh
+
+   --format string               the format of the command output [plain, json] (default "plain")
+   --insecure-sha1-intermediate  allows the use of insecure TLS protocols, such as SHA-1
+   --local                       allows communicating with the server through a unix socket
+   --strict                      will only run commands if the mmctl version matches the server one
+
+mmctl permissions reset
+^^^^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+Reset the given role's permissions to the default settings and overwrite custom settings (only available in Enterprise Edition E10 and E20).
```

Comment:
Same comment about the word "only" here as well.

Reply:
Thank you, @wiersgallak! Fixed.